cypress/integration/articleBodySpec.js

```diff
@@ -0,0 +1,126 @@
+import config from '../support/config/services';
+import {
+  copyrightDataWindow,
+  firstHeadlineDataWindow,
+  firstParagraphDataWindow,
+  firstSubheadlineDataWindow,
+  getElement,
+  placeholderImageLoaded,
+  renderedTitle,
+  visibleImageNoCaption,
+  visibleImageWithCaption,
+  shouldContainText,
+} from '../support/bodyTestHelper';
+
+Object.keys(config)
+  .filter(service => config[service].pageTypes.articles !== undefined)
+  .forEach(service => {
+    describe(`${service} Article Body Tests`, () => {
+      // eslint-disable-next-line no-undef
+      beforeEach(() => {
+        cy.visit(
+          `/${service}/articles/${config[service].pageTypes.articles.asset}`,
+        );
+      });
+
+      it('should render an H1, which contains/displays a styled headline', () => {
+        firstHeadlineDataWindow();
+      });
+
+      it('should render a formatted timestamp', () => {
+        cy.window().then(win => {
+          if (win.SIMORGH_DATA.pageData.metadata.language === 'en-gb') {
```

Comment:
This `if english, else persian` logic doesnt scale to any other services. Is there an issue to refactor this in future? Can this be refactored now?

Reply:
I could think of using switch(service) when we have the article pages for other services.

Reply:
Also not sure whether we will run the e2e tests for all services or few selected services to reduce build time.

Reply:
Are you happy for this to be done when we have the other service article pages?

Reply:
@dr3 is happy to be done this as a part of a separate issue.